### DISCUSSION TITLE
Reputation

### DISCUSSION THREAD

**Discussion by KDaum50** [Vdauphin/HeartsAndMinds#1395] [Question and answer]
Is there a way to check the units Reputation?

**Comment by Vdauphin** [comment]
Hello,
http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#reputation
Cheers

**Reply by KDaum50** [reply]
No like is there like a key bind or something to see what your reputation is at so we can keep track of our progress

**Reply by Vdauphin** [reply]
No because it is unrealistic
And actually, interacting with civilian is one of the core feature of H&M

**Reply by KDaum50** [reply]
ok cool just curious what number do i need to change to to make it that we only respawn on base?

**Reply by Vdauphin** [reply]
Please create a new discussion when asking a new question about an other topic
